vernac/vernacexpr.ml

```diff
@@ -405,11 +405,6 @@ type vernac_control =
      the flag is used to print differently in `-time` vs `Time foo` *)
   | VernacTime of bool * vernac_control CAst.t
   | VernacRedirect of string * vernac_control CAst.t
-  | VernacTimeout of int * vernac_control
-  | VernacFail of vernac_control
+  | VernacTimeout of int * vernac_control CAst.t
+  | VernacFail of vernac_control CAst.t
 
```

Comment:
The linter wants you to remove this empty line.

Reply:
Thanks!